src/Bridges/bridgeoptimizer.jl

```diff
@@ -323,6 +323,25 @@ function MOI.add_constraint(b::AbstractBridgeOptimizer, f::MOI.AbstractFunction,
         return MOI.add_constraint(b.model, f, s)
     end
 end
+function MOI.add_constraints(b::AbstractBridgeOptimizer, f::Vector{F},
+                            s::Vector{S}) where {F <: MOI.AbstractFunction, S <: MOI.AbstractSet}
+    if is_bridged(b, F, S)
+        # We compute `BridgeType` first as `concrete_bridge_type` calls
+        # `bridge_type` which might throw an `UnsupportedConstraint` error in
+        # which case, we do not want any modification to have been done
+        BridgeType = concrete_bridge_type(b, F, S)
+        # `add_constraint` might throw an `UnsupportedConstraint` but no
```

Comment:
Do we have a test to cover this ?

Reply:
To be honestly this was in the non-plural method, and I am not sure that I fully under how to check it.

Reply:
Just write "See `add_constraint` for why we we call `concrete_bridge_type` separately before `add_constraint`" then